demos/iot_config.h

```diff
@@ -45,8 +45,13 @@
  * #define IOT_DEMO_IDENTIFIER         "" */
 
 /* MQTT demo configuration. The demo publishes bursts of messages. */
-#define IOT_DEMO_MQTT_PUBLISH_BURST_COUNT       ( 10 ) /* Number of message bursts. */
-#define IOT_DEMO_MQTT_PUBLISH_BURST_SIZE        ( 10 ) /* Number of messages published in each burst. */
+#define IOT_DEMO_MQTT_PUBLISH_BURST_COUNT    ( 10 )    /* Number of message bursts. */
+#define IOT_DEMO_MQTT_PUBLISH_BURST_SIZE     ( 10 )    /* Number of messages published in each burst. */
+
+/* MQTT library configuration. */
+#ifndef IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES
+    #define IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES    ( 0 )
+#endif
```

Comment:
`iot_mqtt_internal.h` already has this `#ifndef`. Does putting `IOT_BUILD_TESTS` in that file succeed in eliminating the MISRA 20.9 warning?

Reply:
Yes, iot_mqtt_internal.h already has this. I added it here for iot_mqtt_types.h. I know this is not perfect solution.